Standards/scs-0001-v1-sovereign-cloud-standards.md

```diff
@@ -261,7 +288,7 @@ it can be deprecated.
 Obsoletions SHOULD be announced ahead of their execution by setting the
 `deprecated_at` field to a future date and moving the `status` to `Deprecated`.
 This signals current and future implementors
-that the subject matter of the document
+that the subject of the document
```

Comment:
why this change?

Reply:
IMO `subject matter` is redundant, subject (or matter) themselves transport the same meaning.